File: python_programs/calculo_desconto.py
preco = float(input("Digite o preço do produto: "))
desconto = float(input("Digite o percentual de desconto (Ex: Se foi 30,5%, digite apenas 30.5): "))
preco = preco * (1 - (desconto/100))
print("preço final do produto é R$", preco)
print("preço final do produto é R$", "%.2f" %preco) #printf-style String Formatting
print("preço final do produto é R$ {preco:.2f}") #F-Strings

File: javascript_programs/calculoDesconto.js
let preco = parseFloat(prompt("Digite o valor do produto: "))
let desconto = parseFloat(prompt("Digite o percentual de desconto (Ex. Se o desconto for 25,5%, digite apenas 25.5): "))

preco = preco * (1 - (desconto/100))

console.log(`O preço final do produto após o desconto de ${desconto}% é R$${preco.toFixed(2)}`)